bc_obps/reporting/json_schemas/2024/coal_storage/default_ef_custom.json

```diff
@@ -0,0 +1,137 @@
+{
+  "type": "object",
+  "properties": {
+    "coalPurchases": {
+      "type": "array",
+      "title": "Coal Purchases",
+      "items": {
+        "type": "object",
+        "properties": {
+          "annualCoalPurchaseAmount": {
+            "title": "Annual Amount Purchased Coal",
+            "type": "number",
+            "minimum": 0
```

Comment:
Assuming "tonnes" is the unit, should we set that either here or through the UI schema?